packages/home-service/src/resolver.ts

```diff
@@ -1,33 +1,51 @@
 import { Home } from './schema';
+import { HomeHelper } from './helpers';
 
 export const HomeResolver = {
   Query: {
-    // fetch all homeType
     listHomeType(root: any, args: any, ctx: any) {
       return Home.find()
-      .then(response => response)
+      .then( async (response) => {
+        const query = `${HomeHelper.buildGqlQuery(response)}`;
+        return await HomeHelper.getUserDetails(query).then((userDetails: any) => {
+          return HomeHelper.stitchHomeType(response, userDetails.data);
+        });
+      })
       .catch(err => err);
     },
-    // fetch the id from args.id
     getHomeType(root: any, args: any, ctx: any) {
       return Home.findById(args._id)
-      .then(response => response)
+      .then( async (response: HomeType | null) => {
+        if (response !== null) {
+          const query = `${HomeHelper.buildGqlQuery([response])}`;
+          return await HomeHelper.getUserDetails(query).then((userDetails: any) => {
```

Comment:
Don't need an await here. The returned promise will be automatically resolved by GraphQL.

Reply:
Same for other resolvers.

Reply:
Removed `async/await` from the resolvers